cmd/kinflate/README.md

```diff
@@ -25,14 +21,79 @@ control commits.  It encourages forking a configuration,
 customizing it, and occasionally rebasing to capture
 upgrades from the original configuration.
 
+
 ## Installation
 
+`kinflate` is written in Go and can be installed using `go get`:
+
 <!-- @installKinflate @test -->
+```shell
+go get -u k8s.io/kubectl/cmd/kinflate
 ```
-go get k8s.io/kubectl/cmd/kinflate
+
+After running the above command, `kinflate` should get installed in your `GOPATH/bin` directory.
+
+## Usage
+
+This section explains sub-commands of `kinflate` using examples. 
+
+### init 
+`kinflate init` initializes the Kubernetes Application by creating an
+application manifest file named `Kube-manifest.yaml` in current directory. This
+should be run in the same directory where application config resides.
+
+```shell
+kinflate init
 ```
 
+### set nameprefix
+
+`kinflate set nameprefix` updates the nameprefix field in the [manifest].
+`kinflate inflate` uses the prefix to generate prefixed names for Kubernetes resources.
+
+#### examples
```

Comment:
These don't have to be titles, you can just say "Examples:"

Reply:
Done.